gatorgrouper/tests/test_num_group.py

```diff
@@ -0,0 +1,75 @@
+"""Testing number of groups"""
+from utils import parse_arguments
+
+
+def test_check_number_groups_size_zero():
+    """Checking 0 groups"""
+    numgrp = 0
+    student_identifiers = [
+        "maria",
+        "dan",
+        "simon",
+        "jesse",
+        "jon",
+        "michael",
+        "jacob",
+        "kapfhammer",
+    ]
+    student_groups = parse_arguments.check_valid_num_group(numgrp, student_identifiers)
+    assert student_groups
+
+
+def test_check_number_groups_size_one():
+    """Checking 1 group"""
+    numgrp = 1
+    student_identifiers = [
+        "maria",
+        "dan",
+        "simon",
+        "jesse",
+        "jon",
+        "michael",
+        "jacob",
+        "kapfhammer",
+    ]
+    student_groups = parse_arguments.check_valid_num_group(numgrp, student_identifiers)
+    assert student_groups
+
+
+def test_check_number_groups_size_muti():
+    """Checking 1 group"""
```

Comment:
There's a misspelling of `multi` here. :smiley: 

Additionally, the docstring for this method is incorrect.

Lastly, it is probably not good practice to test two different group sizes in one test. This is an ***excellent*** chance to use a single parameterized test case!!

Reply:
The test cases are not the finalized version. @libailong and @mmarconi are responsible for implementing `Hypothesis` which can be used to write parameterized test case with randomized testing input. I think it is probably fine to leave it like that for now since it will be changed later anyway.

Reply:
@Lancasterwu the docstring fix is the only thing left I think then!

Reply:
Thx, just fixed it!